specs/externalLinks/external_links_api_specs.yaml

```diff
@@ -0,0 +1,175 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: External links create api
+paths:
+  /orchestrator/global/externalLinks:
```

Comment:
remove global word from url.

use "-" instead of camel casing in urls (for example : externalLinks should be external-links)